rcl/test/rcl/test_info_by_topic.cpp

```diff
@@ -295,8 +300,8 @@ TEST_F(CLASSNAME(TestInfoByTopicFixture, RMW_IMPLEMENTATION),
 TEST_F(CLASSNAME(TestInfoByTopicFixture, RMW_IMPLEMENTATION),
   test_rcl_get_publishers_subscription_info_by_topic)
 {
-  // This is implemented only in fastrtps currently.
-  if (!is_fastrtps) {
+  // This is implemented only in fastrtps and Cyclone DDS currently.
+  if (!is_fastrtps && !is_cyclonedds) {
     GTEST_SKIP();
   }
   rmw_qos_profile_t default_qos_profile;
```

Comment:
Can you initialize this with https://github.com/ros2/rmw/blob/35ae2ff54317e8e6f30acace2fbaccd57fb2c566/rmw/include/rmw/qos_profiles.h#L90?

Reply:
Fixed in my last commit.